tasks/2017/spread-the-word.md

```diff
@@ -0,0 +1,10 @@
+# GCI Tasks: Spread The Word
+
+## Background
+
+Upvotes and reviews make a big difference in the public perception of
+projects like Zulip. 
+
+## Task description
+
+In this task, we will ask you to rate and review Zulip on some technology sites. Mention that you're doing this task on some stream, and a mentor will send you the instructions. You're free to decide to not do the task after reading the instructions. We expect completing all the tasks will take about 90 minutes. 
```

Comment:
Maybe "You're free to decide to do or not to do the task after reading the instructions."

Reply:
A few minor notes:
* We line wrap to 80 chars to make it easy to see diffs and the output of `git grep`. In emacs I think you can do this with `Meta-q`; it's probably worth finding out what this is for your editor.
* some stream -> should be more specific, like "on foo stream, with topic bar"